sdk/communication/azure-communication-chat/azure/communication/chat/_chat_client.py

```diff
@@ -75,7 +75,7 @@ def __init__(
 
         self._client = AzureCommunicationChatService(
             self._endpoint,
-            authentication_policy=CommunicationUserCredentialPolicy(self._credential),
+            authentication_policy=BearerTokenCredentialPolicy(self._credential, ""),
```

Comment:
Let's check if we need to pass scopes to the constructor. if not we can omit this param.

Reply:
Scopes is not necessary, will remove it now 👍🏻